packages/frontend/src/components/dialogs/DisappearingMessages.tsx

```diff
@@ -57,39 +58,34 @@ function SelectDisappearingMessageDuration({
       />
       <Radio
         key={'eph-1'}
-        label={tx('after_1_minute')}
```

Comment:
So what if the value is already set to "1 minute" in an older version? If they upgrade, none of the radio buttons will be shown as selected. Not ideal IMO. (But also not too bad).